### PR TITLE
Helps 132 components validation

### PR DESCRIPTION
**Pull request by RusGraf**
I added input-able and select-able components validation

### CODE REVIEW

File: ldk/javascript/examples/self-test-loop/src/config/whisper-validation-group.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+import * as whisperValidationTests from '../tests/whisper/whisper-validation';
+
+export const whisperValidationTestGroup = (): TestGroup =>
+  new TestGroup('Whisper Validation', [
+    new LoopTest(
+      'Whisper Validation - Basic Form Validation',
+      whisperValidationTests.testComponentsValidation,
+      20000,
+      `Did form validation worked?`,
```
Comment (cshields-oliveai) [inline ldk/javascript/examples/self-test-loop/src/config/whisper-validation-group.ts:12]: How about "Did the form validate?"

File: ldk/javascript/examples/self-test-loop/src/tests/whisper/utils.ts [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    console.error(error);
+    reject(error);
+  }
+  whisperToClose.close(() => {
+    // do nothing.
+  });
+  reject(new Error('Not rendered correctly.'));
+};
+
+export const resolveRejectButtons = (
```
Comment (cshields-oliveai) [inline ldk/javascript/examples/self-test-loop/src/tests/whisper/utils.ts:40]: This is very similar to `confirmOrDeny` inside of `tests/whisper/whisper-update.ts`. Not necessarily your issue to combine these but perhaps we can at some point (use the one you created in `/utils`)

File: ldk/javascript/examples/self-test-loop/src/tests/whisper/validation/index.ts [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+  RadioGroup,
+  Checkbox,
+  Select,
+} from '@oliveai/ldk/dist/whisper/types';
+import ComponentIds from './componentIds';
+
+const emailRegex = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+const phoneRegex = /^[\+]?[(]?[0-9]{3}[)]?[-\s\.]?[0-9]{3}[-\s\.]?[0-9]{4,6}$/im;
+
+export const validateForm = (
```
Comment (cshields-oliveai) [inline ldk/javascript/examples/self-test-loop/src/tests/whisper/validation/index.ts:18]: cool

**Review by cshields-oliveai — APPROVED** [review]

File: ldk/javascript/examples/self-test-loop/src/tests/whisper/utils.ts (hunk 2 above)
Comment (RusGraf) [reply ldk/javascript/examples/self-test-loop/src/tests/whisper/utils.ts:40]: Yes the idea was to extract this buttons into common place. I just didnt refactored  the whisper update yet.

**Review by epiccoleman — APPROVED** [review]

**Pull request merged**